CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 
 - \[[#287](https://github.com/rust-vmm/vm-memory/pull/287)\] Support for RISC-V 64-bit platform.
 - \[[#299](https://github.com/rust-vmm/vm-memory/pull/299)\] atomic_bitmap: support enlarging the bitmap.
+- \[[#304](https://github.com/rust-vmm/vm-memory/pull/304)\] Implement ReadVolatile and WriteVolatile for TcpStream
```

Comment:
Can you move this to the `Upcoming version` section? We already released 0.16 :(

Reply:
> Can you move this to the `Upcoming version` section? We already released 0.16 :(
> 
> Should be
> 
> ```
> ## Upcoming version
> 
> ### Added
> 
> - \[[#304](https://github.com/rust-vmm/vm-memory/pull/304)\] Implement ReadVolatile and WriteVolatile for TcpStream
> ```

Sorry. I have made the changes accordingly.

Thanks for your detailed correction:)

Reply:
Thank you!